misk/web/modules/log4j/webpack.development.config.js

```diff
@@ -0,0 +1,43 @@
+const miskCommon = require('../../@misk/common/lib');
```

Comment:
I’m tempted to ask you to delete the other modules so that we don’t need to duplicate my pedantic edits over and over

Reply:
Yeah, I'm going to scrap the current log4j and threads since those were essentially just the test apps that came with `FrintJS` and use config as the base exemplar module.